### PR TITLE
CLM: Add Use Cases

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9028

Source: https://w3.suse.de/~mc/SUSEManager/CML-Filter-Use-Cases.txt

### CODE REVIEW

**Comment by jcayouette** [comment]
Already on this?

**Comment by Loquacity** [comment]
> I am currently working on this? Why create a new PR? Admittedly it might be a good idea since my local branch is a little stale(I do not really want to deal with conflicts). I will pull and copy my changes over, you have the same structure. I am currently working on updating the **Reference Book -> Content Lifcycle -> Filters** section and explaining each of the available filter types.

Sorry, that was me misunderstanding what you were working on. Please work in a PR so I can keep up to date with what you're doing.

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+
+This section contains some common examples of content lifecycles.
+Use these examples to build your own personalised implementation.
+
+[#create-proj-for-patch-cycle]
+== Create a Project for a Monthly Patch Cycle
+
+[#create-a-filter]
+=== Create a Filter
```
[the remark this answers is not in the document]
Comment (jcayouette) [reply modules/administration/pages/content-lifecycle-examples.adoc]: I am not sure I agree. Each of the level 3 subheadings here are part of "Creating a Project for a monthly patch cycle? Each of these headings are required sub steps to perform the lead heading action?

**Comment by jcayouette** [comment]
@Loquacity @keichwa I have added your requests Lana, Karl could you take a look at the files and see if there are any glaring issues? Thanks!

**Comment by keichwa** [comment]
> @Loquacity @keichwa I have added your requests Lana, Karl could you take a look at the files and see if there are any glaring issues? Thanks!

Well done!  This was big!  I'll take a closer look, and might spot minor issues.

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,260 @@
+[[content-lifecycle]]
+= Content Lifecycle Management Examples
+
+This section contains some common examples of how you can use content lifecycle management.
+Use these examples to build your own personalised implementation.
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
Use these examples to build your own personalized implementation.
```

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+
+
+[#create-proj-for-patch-cycle]
+== Creating a Project for a Monthly Patch Cycle
+
+
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
A project for a monthly patch cycle consists of:
```
Or something similar. We need an intro sentence, IMO.  (And we probably do not want 3 empty lines before the list starts.)

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+* Creating a filter for a project
+* Adding a filter to a project
+* Applying the filter to a new build environment
+* Excluding a patch from a project
+* Including a patch in a project
+
+
+
+[#create-a-filter]
+=== Create a Filter for the Project
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
=== Creating a Filter for the Project
```
(For consistency.  Lana will probably propose to get rid if the ing form in section titles.)

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+.Procedure: Creating the ``By Date`` Filter
+. In the {productname} {webui}, navigate to menu:Content Lifecycle[Filters] and click btn:[Create Filter].
+
+. In the [guimenu]``Filter Name`` field, type a name for your filter.
+For example, [systemitem]``Exclude patches by date``.
+. In the [guimenu]``Filter Type`` field, select [guimenu]``Patch (issued after)``.
+
+. Select the date and time.
+. Click btn:[Save].
+
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
```
1 empty line too much...

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+Filters determine which metadata will be used to define your channels, as well the packages within those channels.
+====
+
+
+
+[#exclude-a-patch]
+=== Excluding a Patch from the Project
+
+Tests may help you discover issues.
+Once an issue is found, exclude the problem patch released before the **by date** filter.
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
When an issue is found, exclude the problem patch released before the **by date** filter.
```
per SUSE style guide

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+.Procedure: Excluding a Patch
+
+. In the {productname} {webui}, navigate to menu:Content Lifecycle[Filters] and click btn:[Create Filter].
+
+. In the [guimenu]``Filter Name`` field, enter a name for the filter.
+For example, [systemitem]``Exclude openjdk patch``.
+. In the [guimenu]``Filter Type`` field, select [guimenu]``Patch (matches Advisory Name)``.
+
+. In the [guimenu]``Advisory Name`` field, type a name for the advisory.
+For example, [systemitem]``SUSE-15-2019-1807``
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
For example, [systemitem]``SUSE-15-2019-1807``.
```

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+. Detach the ``include`` filter. 
+The filter is included by default.
+
+. Complete updating your filters.
+. Rebuild the project to create a new environment with patches for 1 September.
+
+
+
+[#enhance-project-with-livepatching]
+== Enhance a Project with Livepatching
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
== Enhance a Project with Live Patching
```

File: modules/administration/pages/content-lifecycle-examples.adoc [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+This patch contains ``kernel-default-4.12.14-150.17.1-x86_64``.
+
+You need to exclude all patches which contain a higher version of ``kernel-default``.
+
+Procedure: Excluding Packages with a Higher Kernel Version
+
+. In the {productname} {webui}, navigate to menu:Content Lifecycle[Filters], and click btn:[Create Filter].
+
+. In the [guimenu]``Filter Name`` field, type a name for your filter.
+For example, [systemitem]``Exclude kernel greater than 4.12.14-150.17.1``
```
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc]:
```suggestion
For example, [systemitem]``Exclude kernel greater than 4.12.14-150.17.1``.
```

File: modules/administration/pages/content-lifecycle-examples.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/administration/pages/content-lifecycle-examples.adoc]: Yeah, base form of the verb in section titles, gerund form in procedure titles. Sorry!

File: modules/administration/pages/content-lifecycle-examples.adoc (hunk 9 above)
Comment (Loquacity) [reply modules/administration/pages/content-lifecycle-examples.adoc]: @keichwa we can always rely on your to find the missing periods! Nice work! :heart:

**Comment by Loquacity** [comment]
> @Loquacity @keichwa I have added your requests Lana, Karl could you take a look at the files and see if there are any glaring issues? Thanks!

Looks like some of mine are still outstanding.

File: modules/administration/pages/content-lifecycle-examples.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/content-lifecycle-examples.adoc]: Oooh, nice catch!

**Comment by Loquacity** [comment]
Also: I would recommend squashing these commits before you merge ...

**Pull request merged**